modules/cnf-du-workload-partitioning-pod-mutation.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/cnf-provisioning-and-installing-a-distributed-unit.adoc
+
+[id="cnf-du-workload-partitioning-pod-mutation_{context}"]
+
+= Workload partitioning and pod mutation
+
+The workload partitioning feature modifies pods that are annotated with `target.workload.openshift.io/<workload-type>`
+that are in a namespace that contains the `workload.openshift.io/allowed: <workload-type>` annotation.
+
+For example, a pod with the following CPU request specified would be mutated:
+
+[source,yaml]
+----
+requests:
+  cpu:
+    400m
+----
+
+The requested CPU resource is replaced with a management cores resource:
+
+[source,yaml]
+----
+requests:
+  management.workload.openshift.io/cores: 400
+----
+
+The value of `resources.workload.openshift.io/{container-name}` is calculated as:
+
+----
+shares == (request_in_millis * 1024) /1000
+----
+
+This results in a value of 409 in this example.
+
+An annotation is added with the same value:
+
+[source,yaml]
+----
+annotations:
+  resources.workload.openshift.io/{container-name}: {"cpushares": 409}
+----
+
+The new request value and annotation value are scaled up by 1000 from the original CPU request input because
+opaque resources do not support units or fractional values. Note in the example above the request is in
+milli-cores (400m) and the modified value is unitless.
+
+Pods are not changed in a way that changes their Quality-of-Service (QoS) class. For example, the feature does not strip CPU requests unless the pod also has memory requests, because if we mutate the pod so that it has no CPU or memory requests the Quality-of-Service class of the pod would be changed automatically. In this case, the `workload.openshift.io/warning` annotation includes a message explaining that the partitioning instructions were ignored. Any pod that is already BestEffort is annotated using `2` as the value so that CRI-O has an indicator to configure the CPU shares as BestEffort.
```

Comment:
Please remove the hyphens from "Quality of Service."  That way, your work matches the [topology mgr page](https://docs.openshift.com/container-platform/4.6/scalability_and_performance/using-topology-manager.html) in the Perf & Scale section.

"strip" is still eye catching.  A Ctrl+F for "strip" on the following page has no hits.  A search for "remove" has 6. I looked at https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#mutatingadmissionwebhook

If you have a resource that uses "strip" and you expect most customers have read it, please share.

Regarding the QoS information:
* Please enclose BestEffort and Guaranteed in backticks. The context suggests to me that you are using them as a literal values and backticks are the markup according to the [quick markup reference](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference).
* The last bullet, "Pods...do not mutate" is surprising.  My understanding is that some software component, such as an admission controller or webhook mutates the pod.  Maybe "Pods...are not mutated".